src/main/java/edu/ie3/datamodel/io/factory/timeseries/IconTimeBasedWeatherValueFactory.java

```diff
@@ -35,16 +35,16 @@ public class IconTimeBasedWeatherValueFactory extends TimeBasedWeatherValueFacto
   private static final String WIND_VELOCITY_U = "u131m";
   private static final String WIND_VELOCITY_V = "v131m";
 
-  public IconTimeBasedWeatherValueFactory(TimeUtil timeUtil) {
-    super(timeUtil);
+  public IconTimeBasedWeatherValueFactory() {
+    super();
   }
 
-  public IconTimeBasedWeatherValueFactory(String timePattern) {
-    super(timePattern);
+  public IconTimeBasedWeatherValueFactory(TimeUtil timeUtil) {
+    super(timeUtil);
```

Comment:
Since `DateTimeFormatter` is now the only parameter to the `TimeUtil` constructor, it seems like this constructor is now obsolete given the other one below?